src/psyclone/transformations.py

```diff
@@ -2836,17 +2837,17 @@ class ACCKernelsTrans(RegionTrans):
 
     For example:
 
-    >>> from psyclone.parse import parse
+    >>> from psyclone.parse.algorithm import parse
```

Comment:
Since we're here, it's worth doing this properly. Could you have a quick look and see whether you can get doctest to pass for this docstring (`python -m doctest transformations.py`). If it gets complicated we'll leave it as it's not really your problem.